tests/test_aeo_dataset_registration.py

```diff
@@ -86,25 +91,100 @@ def replace_dataset_path(dataset_config_file, dataset_path):
     logger.info("Replaced dataset path in %s with %s", dataset_config_file, src_data)
 
 
-### set up test for AEO data ###
-def test_aeo_datasets(make_test_project_dir, make_test_data_dir):
+### set up tests for AEO data ###
+def test_aeo_datasets_registration(make_test_project_dir, make_test_data_dir):
     if "test_aeo_data" not in os.listdir(make_test_data_dir):
         print("test_invalid_datasets requires the dsgrid-test-data repository")
         sys.exit(1)
 
     datasets = os.listdir(make_test_data_dir / "test_aeo_data")
-    datasets = [datasets[0]]  # <-- test only End Uses for now
-    # TODO: End Use Growth Factors will be tested in another PR
     for i, dataset in enumerate(datasets, 1):
         print(f">> Registering: {i}. {dataset}...")
         data_dir = make_test_data_dir / "test_aeo_data" / dataset
-        with TemporaryDirectory() as tmpdir:
-            base_dir = Path(tmpdir)
-            print(f"temp_registry created: {base_dir}")
-            manager = make_registry_for_aeo(
-                base_dir,
-                make_test_project_dir,
-                dataset,
-                dataset_path=data_dir,
-                include_datasets=False,
+
+        shutil.copyfile(data_dir / "load_data.csv", data_dir / "load_data_original.csv")
+
+        print("1. normal registration: ")
```

Comment:
Do the print statements help debug on errors? Would it be better to use logging debug statements instead? pytest eats stdout by default and it's easy to enable debug on demand.